src/OrchardCore.Modules/OrchardCore.Facebook/Services/FacebookCoreService.cs

```diff
@@ -46,32 +46,32 @@ public async Task UpdateSettingsAsync(FacebookCoreSettings settings)
             await _siteService.UpdateSiteSettingsAsync(container);
         }
 
-        public Task<ImmutableArray<ValidationResult>> ValidateSettingsAsync(FacebookCoreSettings settings)
+        public Task<IEnumerable<ValidationResult>> ValidateSettingsAsync(FacebookCoreSettings settings)
```

Comment:
Out of curiosity, why these changes?

Reply:
I don't see the point of requiring an immutable array on the interface. I remember you had explained why you use them, please remind me.